lm75bd/lm75bd.h

```diff
@@ -5,7 +5,8 @@
 #include <stdint.h>
 
 /* LM75BD I2C Device Address */
-#define LM75BD_OBC_I2C_ADDR /* Define the address here */
+#define LM75BD_OBC_I2C_ADDR 0b1001111U
```

Comment:
Address is correct. Hexadecimal is usually used as binary can easily become unreadable as numbers get larger.